active/corsair.py

```diff
@@ -0,0 +1,54 @@
+"""
+This is an adaptation of Corsair Scan (https://github.com/Santandersecurityresearch/corsair_scan) as a ZAP active scan script.
+This script will resend requests to all the sites captured in ZAP, injecting different origins. Then, it will read the value of Access-Control-Allow-Origin and based on that, it
```

Comment:
`This script will resend requests to all the sites captured in ZAP` is not accurate, it will be just on the sites being scanned.